control-plane/pkg/apis/internals/kafka/eventing/v1alpha1/consumer_group_validation_test.go

```diff
@@ -62,28 +62,6 @@ func TestConsumerGroup_Validate(t *testing.T) {
 			},
 			wantErr: true,
 		},
-		{
```

Comment:
Can we keep at lease one test that verifies that the ConsumerTemplateSpec is validated as part of the ConsumerGroup validation?